Resources/Prototypes/Recipes/Lathes/tools.yml

```diff
@@ -134,3 +134,13 @@
   materials:
     Steel: 500
     Plastic: 250
+
+- type: latheRecipe
+  id: HandheldGPSBasic
+  icon: Objects/Devices/gps.rsi/icon.png
+  result: HandheldGPSBasic
+  completetime: 2000
+  materials:
+    Steel: 800
+    Plastic: 400
```

Comment:
Isn't plastic very rare? Not sure how valuable the gps is compared to what it requires to get but I guess we'll see.

Reply:
I'll just remove the plastic since the steel and glass input enough is a decent bit.

Reply:
> Delete empty server system

Yeah, I feel real dumb for that one. overlooked it.
Made the changes and everything is still working, thanks.